<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1121</width>
      <height>601</height>
     </rect>
    </property>
    <widget class="QWidget" name="Conference">
     <attribute name="title">
      <string>Team By Conferences</string>
     </attribute>
     <widget class="QTableView" name="ConferenceTableView">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>941</width>
        <height>551</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="TeamsComboBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="AFLCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>AFL</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="NFLCheckBox">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>NFL</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="BothCheckBox">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Both</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1120</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
